dao/src/main/java/org/wso2/testgrid/dao/uow/TestPlanUOW.java

```diff
@@ -61,6 +61,7 @@ public TestPlan persistTestPlan(TestPlan testPlan) throws TestGridDAOException {
             persisted.setScenarioTestsRepository(testPlan.getScenarioTestsRepository());
             persisted.setInfrastructureRepository(testPlan.getInfrastructureRepository());
             persisted.setDeploymentRepository(testPlan.getDeploymentRepository());
+            persisted.setConfigChangeSetRepository(testPlan.getConfigChangeSetRepository());
```

Comment:
We need to do this for TestPlan#clone operation as well - https://github.com/wso2/testgrid/blob/aa462b763be00a4a8169e39daaa1152fc184d77d/common/src/main/java/org/wso2/testgrid/common/TestPlan.java#L381